packages/grafana-schema/src/common/dimensions.cue

```diff
@@ -0,0 +1,25 @@
+package common
+
+BaseDimensionConfig: {
+  field?: string
+  fixed: string | number
```

Comment:
I see it's overwritten in veneer to be a generic type, so it can be `fixed: _` here, which will generate it as `unknown` and you won't need to omit it when extending.

Reply:
I couldn't get this to work. The problem is the fixed property exists both on the veneer for the `BaseDimensionConfig` and on `raw.ScaleDimensionConfig` where it's inherited from `raw.BaseDimensionConfig`